packages/appsync-emulator-serverless/lambda/util.js

```diff
@@ -0,0 +1,40 @@
+const log = require('logdown')('appsync-emulator:lambdaRunner');
+
+const parseErrorStack = error =>
+  error.stack
+    .replace(/at /g, '')
+    .split('\n    ')
+    .slice(1);
+
+const sendOutput = output => {
+  console.log(output);
+  process.send({ type: 'success', output }, process.exit);
+};
+const sendErr = err => {
+  let error;
+  console.log(err);
```

Comment:
can we remove this? Or at least have it invoked by log ?

Reply:
Forgotten logs :)